test/import/import-cli-config-sync.test.ts

```diff
@@ -0,0 +1,95 @@
+import * as os from 'os';
+import * as path from 'path';
+import * as fs from 'fs-extra';
+import * as yaml from 'yaml';
+import { readConfigSync, addImportToConfig } from '../../src/config';
+import { Language } from '../../src/import/base';
+import { ManifestObjectDefinition } from '../../src/import/crd';
+import { importDispatch } from '../../src/import/dispatch';
+
+const configFilePath = path.join(__dirname, 'configOutput');
+const jenkinsCRD = 'https://raw.githubusercontent.com/jenkinsci/kubernetes-operator/master/deploy/crds/jenkins.io_jenkins_crd.yaml';
+
+async function withTempFixture(data: any, test: (fixture: string, configFile: string, cwd: string) => Promise<void>) {
+  const tempDir = fs.mkdtempSync(path.join(os.tmpdir(), 'cdk8s-import-test'));
+  const fixture = path.join(tempDir, 'fixture.yaml');
+  try {
+    if (Array.isArray(data)) {
+      fs.writeFileSync(fixture, data.map(d => yaml.stringify(d)).join('\n---\n'));
+    } else {
+      fs.writeFileSync(fixture, yaml.stringify(data));
+    }
+    await test(fixture, configFilePath, tempDir);
+  } finally {
+    fs.removeSync(tempDir);
+    // clean up cdk8s.yaml file, reseting back to template:
+    const defaultConfig = yaml.parse(fs.readFileSync(path.join(configFilePath, 'cdk8s-template.yaml'), 'utf-8'));
+    await fs.outputFile(path.join(configFilePath, 'cdk8s.yaml'), yaml.stringify(defaultConfig));
+  }
+}
+
+test('reading default config file works', async () => {
+
+  // clean up cdk8s.yaml file, reseting back to template:
+  const defaultConfig = yaml.parse(fs.readFileSync(path.join(configFilePath, 'cdk8s-template.yaml'), 'utf-8'));
+  await fs.outputFile(path.join(configFilePath, 'cdk8s.yaml'), yaml.stringify(defaultConfig));
+  const config1 = readConfigSync(configFilePath);
```

Comment:
Ah, is this why you added `configFilePath` to the `ImportOptions`? 

In general, whenever you find yourself adding production code solely for the sake of unit tests, think twice, and then another time :). In some *rare* cases its ok, but I don't think we are there.

 Normally, this just means you are testing the wrong thing. For example here you are testing a code path doesn't exist in production, so its not needed.